stories/styles/color/color.stories.tsx

```diff
@@ -15,7 +15,6 @@ const {
   background,
   divider,
   text,
-  type,
```

Comment:
Is this `type` that's been replaced with `mode`? If so, just curious why you removed it rather than replacing it - are we not currently using it in any stories, perhaps?

Reply:
Yeah! I don't know if you remember in our pairing, but this was never being used.